designer/client/page-edit.js

```diff
@@ -25,10 +26,19 @@ export class PageEdit extends React.Component {
       section: page?.section ?? "",
       isEditingSection: false,
       errors: {},
+      toggle: false,
     };
     this.formEditSection = React.createRef();
   }
 
+  async componentDidMount() {
+    const result = await new ToggleApi().fetchToggles();
+
+    this.setState({
+      toggle: result["featureEditPageDuplicateButton"] == true,
```

Comment:
pedantic, no need to action this, maybe better named allow/show/disable/enableFeatureName

Reply:
(I can't tell if toggle is turning something on or off)